docs/SPEC.md

```diff
@@ -1780,36 +1780,36 @@ Example:
 
     histogramQuantile(quantile:0.9)  // compute the 90th quantile using histogram data.
 
-#### LinearBuckets
+#### LinearBins
 
-LinearBuckets produces a list of linearly separated floats.
+LinearBins produces a list of linearly separated floats.
 
-LinearBuckets has the following properties:
+LinearBins has the following properties:
 
 * `start` float
     Start is the first value in the returned list.
 * `width` float
-    Width is the distance between subsequent bucket values.
+    Width is the distance between subsequent bin values.
 * `count` int
-    Count is the number of buckets to create.
+    Count is the number of bins to create.
 * `inifinity` bool
-    Infinity when true adds an additional bucket with a value of positive infinity.
+    Infinity when true adds an additional bin with a value of positive infinity.
     Defaults to `true`.
 
-#### LogrithmicBuckets
+#### LogrithmicBins
```

Comment:
Typo: I think the correct spelling is `logarithmic`, unless this is an alternative spelling that I don't know about.